app/api/business/questions_business.py

```diff
@@ -32,11 +32,14 @@ def get_counts(brief_id, questions=None, answers=None):
     }
 
 
-def get_question(brief_id, question_id):
+def get_question(current_user, brief_id, question_id):
     brief = briefs.find(id=brief_id).one_or_none()
     if not brief:
         raise NotFoundError("Invalid brief id '{}'".format(brief_id))
 
+    if not briefs.has_permission_to_brief(current_user.id, brief.id):
+        raise UnauthorisedError('Unauthorised to publish answer')
+
     question = brief_question_service.find(id=question_id).one_or_none()
```

Comment:
This should include `brief_id` in the find - we're now confirming that the user has permission on the brief, but not that this question id is associated with the brief. Currently `/api/2/brief/XXX/question?questionId=YYY' will return any question id (`YYY`) if `XXX` is a brief you have permission on.

Reply:
fixed